src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs

```diff
@@ -134,6 +134,45 @@ public void TestReflectInitializationViaSubscription3()
             }
         }
 
+        /// <summary>
+        /// Test to make sure we get both request and response events.
+        /// </summary>
+        [Fact]
```

Comment:
Please add this as an outerloop test.  The remote end points you are testing against are subject to Internet flaxiness.

Then run this PR with Outerloop against NETFX (.NET Framework)